OxfordBodleian/Juel-Jensen/BDLaethe37.xml

```diff
@@ -33,28 +36,30 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                             <idno>Juel-Jensen 11</idno>
                         </altIdentifier>
                     </msIdentifier>
-                    
+
                     <msContents>
                         <summary/>
-                        
+
                         <!--<msItem xml:id="ms_i1">
-                      
+
                         </msItem>-->
-                        
+
                     </msContents>
-                    
+
                     <physDesc>
                         <objectDesc form="Codex">
                             <supportDesc>
                                 <support>
                                     <material key="parchment"/>
+                                    <p>Holes, from the parchment making process, are present in the leaves, e.g. <locus target="#77 #83 #84 #92"/> others have been repaired by sewing, e.g.
+                                    <locus target="#2 #9 #58 #60 #61 #72 #95"/>.</p>
                                 </support>
                                 <extent>
                                     <measure unit="leaf">96</measure>
                                     <measure unit="leaf" type="blank"></measure>
                                     <!--<locus target=""/>-->
-                                    <measure type="weight" unit="g">326</measure><!-- without māḫdar and modern notes placed inside taken out-->
-                                    <measure type="weight" unit="g">544</measure><!--with māḫdar, without modern notes-->
+                                    <measure type="weight" unit="g">326>without slip case</measure><!-- without māḫdar and modern notes placed inside taken out-->
```

Comment:
encoding problem (one > too many). I think we might settle now whether we want these double weight measures to be included in the catalogue? In any case, the information on the presence or absence of the slip case might be better in a `<note>` directly after the respective measurement.

Reply:
I think that for the catalogue the most important information is the presence of the slip case while its weight might not be fundamental.

Reply:
It seems reasonable to me to include the weight of the manuscripts itself, without possible accompanying documents/slip cases (this second measurement could stay commented out in the records). That is already more information on weight than there is anywhere else ;)